### DISCUSSION TITLE
Printing Grid Lines in Tables

### DISCUSSION THREAD

**Discussion by Dwindle2234** [TriliumNext/Trilium#3769] [Q&A]
What's the trick to get table grid lines to print? I've tried setting colours to cells/styles to cells/table - all look okay on screen but on printing you only get the very top border line and sometimes the left most line actually printing. Also tried different printers to no avail.

**Comment by zadam** [comment]
Not sure, but printing into PDF preserves the borders: 
<img width="814" alt="image" src="https://user-images.githubusercontent.com/617641/227798218-4e276af5-f6a6-4f2f-a319-dd3e68e2a1b6.png">

I think printers sometimes do such optimizations to save ink. I guess try printing into PDF and then print the PDF itself to see if it makes any difference.

**Reply by Dwindle2234** [reply]
Thanks - I did try into PDF but I got the same issue (Microsoft Print to PDF driver).

**Reply by hulmgulm** [reply]
On my printer, the lines are very thin. I thought about to create a "print theme", but the border width is not consumed from CSS but hardcoded to 1px in element style although the CSS is setting it to 1px too. Can this get changed?

**Reply by zerebos** [reply]
You can override inline styles with `!important`
